src/app/components/footer/footer.html

```diff
@@ -0,0 +1,3 @@
+<div class="container">
```

Comment:
Why there is DIV? You should use FOOTER tag.

Reply:
Yoy can aprove and if you will want fix it)

Reply:
Okay 😊